test/associations_test.rb

```diff
@@ -48,6 +48,22 @@ def test_multiple_associations
         .must_equal expected
     end
 
+    def test_nested_associations
+      peggy = Person.new
+      roger = Person.new(id: 2, name: 'Roger', occupation: 'Chief')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.